packages/core/src/tools/tool-registry.ts

```diff
@@ -378,6 +403,19 @@ function _sanitizeParameters(schema: Schema | undefined, visited: Set<Schema>) {
       }
     }
   }
+
+  // Handle enum values - Gemini API only allows enum for STRING type
```

Comment:
Hmm we didn't support this earlier? Typically the genai SDK will do the translation for this